projects/ngx-prx-styleguide/src/public_api.ts

```diff
@@ -47,6 +47,8 @@ export { ImageModule } from './lib/image/image.module';
 export { ModalModule } from './lib/modal/modal.module';
 export { ModalService } from './lib/modal/modal.service';
 
+export { PagingModule } from './lib/paging/paging.module';
```

Comment:
I think this is all I need to export my paging component, but when I run Publish with this styleguide copied in locally and import the PagingModule into SharedModule, the `prx-paging` component isn't being recognized. I used the demo app to verify the export, which we should probably clean up and get rid of, but it was convenient for checking this public_api.